doc/troubleshooting.adoc

```diff
@@ -95,3 +95,19 @@ If you have installed `kubectl_kadalu` package, then you can do below to get the
 ----
 $ kubectl kadalu logs
 ----
+
+
+=== Quota of PVCs
+
+`kadalu` uses simple-quota feature of glusterfs, which is present only in kadalu storage releases of glusterfs.
+
+As this is a new feature of glusterfs, there is possibilities where an user can hit a bug which is stopping the usage in in production. Hence, we have provided an option to disable quota limit check on PVCs of a particular storage pool. Please use below steps to get this working.
```

Comment:
May be a dumb question, does it effectively mean 1 vol : 1 pv?

Reply:
No. Multiple PVs, but PVs can overuse the storage than they are provisioned. This is only a workaround to unblock the Production issue.

Reply:
Ack, in a way I was also getting to that point.

I feel there should be a note that this section will be revised by next release (or at the earliest possible). 

I can already think of cases where it'll create confusion to users and debugging burden on us.

Please don't mind, if it hadn't been a prod issue, I would've strictly nack'd it. I guess you'll also feel the same😅